### PR TITLE
Emit initial auction event

### PR DESCRIPTION
**Pull request by matthewlane**
* Generate a random auction id
* Return id as string to match expected schema type
* Remove redundant param from BID_RESPONSE event

### CODE REVIEW

File: src/adaptermanager.js [hunk 1]
```diff
@@ -26,7 +26,14 @@ function getBids({ bidderCode, requestId, bidderRequestId, adUnits }) {
 }
 
 exports.callBids = ({ adUnits, cbTimeout }) => {
-  const requestId = utils.getUniqueIdentifierStr();
+  const requestId = utils.generateAuctionId();
+
+  const auctionInit = {
+    timestamp: Date.now(),
+    refererUrl: document.referrer,
```
Comment (mkendall07) [inline src/adaptermanager.js]: We don't need to pass around `referrer` here. Let's remove that.

File: src/adapters/analytics/AnalyticsAdapter.js [hunk 2]
```diff
@@ -93,6 +94,10 @@ export default function AnalyticsAdapter({ url, analyticsType, global, handler }
     events.on(BID_TIMEOUT, args => this.enqueue({ eventType: BID_TIMEOUT, args }));
     events.on(BID_WON, args => this.enqueue({ eventType: BID_WON, args }));
     events.on(BID_ADJUSTMENT, args => this.enqueue({ eventType: BID_ADJUSTMENT, args }));
+    events.on(AUCTION_INIT, args => {
+      args.memberId = config.options.memberId;  // from enableAnaltyics config object
```
Comment (mkendall07) [inline src/adapters/analytics/AnalyticsAdapter.js]: Instead of passing a specific param, shouldn't we send all the `options` (since they will vary)?

Comment (matthewlane) [inline src/adapters/analytics/AnalyticsAdapter.js]: Right, fixed

File: src/adaptermanager.js (hunk 1 above)
Comment (matthewlane) [inline src/adaptermanager.js]: Removed. `referrer` is listed in the schema though, do we need to pass that from somewhere else?

**Comment by mkendall07** [comment]
LGTM

**Comment by rmloveland** [comment]
@mkendall07 It doesn't look to me like this change affects user-facing, documented things.  Is that correct?

**Comment by mkendall07** [comment]
Correct

**Pull request merged**